### DISCUSSION TITLE
How to attach a debugger?

### DISCUSSION THREAD

**Discussion by Lanius-collaris** [celzero/firestack#71] [Q&A]

**Comment by ignoramous** [comment]
From the Android Studio? I guess it'd be the same as attaching debugger to a native Android project?

The unstripped AAR file is got at `com.github.celzero:firestack:<commit>:debug@aar` ([ex](https://github.com/celzero/rethink-app/blob/cc4e55b38ec7c9406f0bb61b6777cb4e959f775b/app/build.gradle#L256)).

Personally, I have been reliant on the good ol' print statements and golang's Test framework.
